pkgs/applications/networking/cluster/crc/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, git
+, stdenv
+, testers
+, crc
+, runtimeShell
+, coreutils
+}:
+
+buildGoModule rec {
+  pname = "crc";
+  version = "2.4.1";
+
+  src = fetchFromGitHub {
+    owner = "code-ready";
+    repo = "crc";
+    rev = "v${version}";
+    sha256 = "4ckHvIswVwECAKsa/yN9rJSqZb04ZP1Zomq+1UdT1OY=";
+    # makefile calculates git commit and needs the git folder for it
+    leaveDotGit = true;
+  };
```

Comment:
Do we really need this or can we just hardcore it with version?

Reply:
We technical can use the `COMMIT_SHA` env var, the default behaviour of the build system is to find the git commit from the `.git`

Reply:
I would suggest to just hardcode COMMIT_SHA to version to keep the fetcher purer.